### ISSUE TITLE
Huawei

### ISSUE THREAD

**Issue by Pervy16** [Vita3K/Vita3K-Android#225]
Crashes when trying to run any game

Huawei p10 plus
Android 9
EMUI 9.1.0
6gb ram
128gb rom
Kirin 960
Mali g71

**Comment by vxpn** [comment]
I have the same my device is: huawei Nova 7i
Android 10
EMUI 12.0.0
8gb ram
128gb rom
Kirin 810
Mali-g52

**Comment by AKJIN** [comment]
Also same problem in my Honor 9xpro
6gb ram 256 rom
Kirin 810
GPU Mali g52

**Comment by Emylian-bit** [comment]
Yeah Huawei devices aren't supported due to Kirin. We have to wait for the dev to fix this

**Comment by AKJIN** [comment]
Thankyou for your replySent from my Huawei phone-------- Original message --------From: Emylian-bit ***@***.***>Date: Sun, 19 Feb, 2023, 3:58 PMTo: Vita3K/Vita3K-Android ***@***.***>Cc: AKJIN ***@***.***>, Comment ***@***.***>Subject: Re: [Vita3K/Vita3K-Android] Huawei (Issue #225)
Yeah Huawei devices aren't supported due to Kirin. We have to wait for the dev to fix this
